packages/itwin/grouping-mapping-widget/src/components/Properties/GroupColorToggle.tsx

```diff
@@ -22,45 +25,52 @@ export const GroupColorToggle = ({
   group,
   ...rest
 }: GroupColorToggleProps) => {
-  const [isLoading, setIsLoading] = useState<boolean>(false);
   const { iModelConnection } = useGroupingMappingApiConfig();
   if (!iModelConnection) {
     throw new Error("This component requires an active iModelConnection.");
   }
   const { showGroupColor, setShowGroupColor } = usePropertiesContext();
-  const { data: hiliteIdsResult } = useGroupKeySetQuery(group, iModelConnection, showGroupColor);
+  const { data: hiliteIdsResult, isFetched, isFetching } = useGroupKeySetQuery(group, iModelConnection, showGroupColor);
+  const isMounted = useIsMounted();
+
+  const { mutate: visualizationMutate, isLoading: isVisualizing } = useMutation({
```

Comment:
`mutateVisualization`, not `visualizationMutate`.

Reply:
Changed